app/views/moderations/mod.html.erb

```diff
@@ -71,7 +71,7 @@
       <a href="/resource_admin/articles/<%= @moderatable.id %>" data-no-instant><%= t("views.moderations.actions.edit.resource_admin") %></a> |
       <a href="<%= admin_article_path(@moderatable.id) %>" data-no-instant><%= t("views.moderations.actions.edit.article_admin") %></a></h3>
   <% elsif current_user.super_admin? && @moderatable.class.name == "Comment" %>
-    <h3> <a href="/admin/comments/<%= @moderatable.id %>" data-no-instant><%= t("views.moderations.actions.edit.comment_admin") %></a> |
+    <h3> <a href="/admin/content_manager/comments/<%= @moderatable.id %>" data-no-instant><%= t("views.moderations.actions.edit.comment_admin") %></a> |
```

Comment:
This is a small fix to a URL that had been broken for a while within comment mod change, not directly related to this fix but along for the ride.